src/services/crawl-account/crawl_account.service.ts

```diff
@@ -301,6 +301,10 @@ export default class CrawlAccountService extends BullableService {
         base_denom: string;
         last_updated_height: number;
       }[] = [];
+      const resetBalanceList: {
```

Comment:
nếu là reset lại tất thì tạo ra luôn ở đây, ko cho vào vòng for kia nữa

resetAccounts = accounts.filter(acc => addressesWithIds.has(acc)).map(acc => {
  account_id: addressesWithIds[acc],
  last_updated_height: account.last_updated_height,
})

Reply:
thêm luôn cái keyBy vào cái list addressesWithIds rồi thay cái câu find

Reply:
Em sua va chay lai thu roi anh nhe
![image](https://github.com/aura-nw/horoscope-v2/assets/49814372/4d90a5e2-110b-4147-ba31-bdb485f8dd8e)
Sau update
![image](https://github.com/aura-nw/horoscope-v2/assets/49814372/b1d2f088-cc32-49bd-b675-637418e1ad43)